tutorial_import_products_inline.md

```diff
@@ -0,0 +1,95 @@
+# **Import Products From Inline Source Tutorial**
+
+## Let's get started
+
+The inline importing is a convenient way to make bulk changes in a catalog:
+
+- import some amount of products, no more than 100 at a time,
+- update some products set,
+- make fast and frequent changes of products quantity or price or any other field.
+
+To import your products into a catalog inline you should prepare the ```product_inline_source``` object which is a set
+of products. Each product should be provided in a JSON format as single line (one product per line with line breaks as a
+delimiter). An example of a product in JSON format can be found in
+a [Retail API documentation](https://cloud.google.com/retail/docs/upload-catalog#json-format)
```

Comment:
To find an example of a product in JSON format, refer to
 the [Retail API documentation](https://cloud.google.com/retail/docs/upload-catalog#json-format)